CHANGELOG.adoc

```diff
@@ -1,5 +1,38 @@
 = Changelog
 
+== v2025.8.0 (work in progress)
+
+=== Shapes
+
+
+
+=== Architectural decision records
+
+
+
+=== Deprecation warning
+
+
+
+=== Breaking changes
+
+
+
+=== Dependency update
+
+
+
+=== Bug fixes
+
+
+
+=== Improvements
+
+- https://github.com/eclipse-sirius/sirius-web/issues/5103[#5103] [sirius-web] Make `ExpandAllTreeItemContextMenuContribution` reusable by downstream applications.
+
+
+
+
```

Comment:
You could try instead to create you PR against the cooldown branch. We have just started creating cooldown branch to allow the integration of PRs while we are freezing the master branch so it's normal if you didn't know about this. Don't do it now, the cooldown branch will be deleted soon :)

Reply:
I saw the 2025.6.0 bump on main and assumed the release was already done. That's why I didn't base my branch on cooldown.
Next time I'll check if cooldown has been merged in main instead